benchmark/benchmark17_test.go

```diff
@@ -0,0 +1,85 @@
+// +build go1.7
+
+package benchmark
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/benchmark/stats"
+)
+
+func BenchClientUnary(b *testing.B, maxConcurrentCalls, reqSize, respSize int) {
+	grpc.EnableTracing = true
+	runUnary(b, maxConcurrentCalls, reqSize, respSize)
+}
+
+func BenchClientStream(b *testing.B, maxConcurrentCalls, reqSize, respSize int) {
+	grpc.EnableTracing = true
+	runStream(b, maxConcurrentCalls, reqSize, respSize)
+}
+
+func BenchClientUnaryNoTrace(b *testing.B, maxConcurrentCalls, reqSize, respSize int) {
```

Comment:
Add one more parameter `tracing` to `BenchClientUnary` and remove this function?

We could probably add a parameter for Unary and Streaming as well. But let's not do that now. Since Unary and Streaming RPC benchmarks can be different.